### DISCUSSION TITLE
Alert Action to a rest  API

### DISCUSSION THREAD

**Discussion by wtakang** [Security-Onion-Solutions/securityonion#2834] [Unsupported Versions] [answered]
Howdy Folks, 
Just wondering if this is possible. I am testing out SO and one of the requirements is being able to feed items from a drill down into a REST API
please advice

**Comment by weslambert** [comment, marked as answer]
HI @wtakang ,

I've wanted something like this myself (webhook, etc), it just hasn't been a priority.  One option would be to customize the alert action URL to point to your API (if it accepts HTTP GETs), and use it that way for the time being.  Otherwise, you may consider using TheHive's webhook functionality for higher fidelity alerts, etc pushed to TheHive as cases by analysts, etc -- I've done a bit of automation in the past using this, and it has worked fairly well.  Another option would be to use Elastalert with a custom alerter, or the built in alerter types like HTTP POST,  etc.

This is being tracked for consideration in https://github.com/Security-Onion-Solutions/securityonion/issues/2861.
